includes/promoted-jobs/class-wp-job-manager-promoted-jobs-notifications.php

```diff
@@ -151,7 +151,7 @@ private function get_notification_url() {
 	 * @return array The data to send.
 	 */
 	private function get_notification_data() {
-		$site_url = home_url();
+		$site_url = home_url( '', 'https' );
 		$feed_url = rest_url( '/wpjm-internal/v1/promoted-jobs' );
```

Comment:
I think we need to have the `rest_url` in the `https` scheme as well, no?

```suggestion
		$feed_url = rest_url( '/wpjm-internal/v1/promoted-jobs', 'https' );
```

Reply:
Probably it would be a good idea to do that for all the endpoints we are sending to WPJM (I am thinking on the get job data and verify endpoints).

Reply:
**I think** we're already doing that on other `home_url` and `rest_url` that we send to WPJMCOM...but yeah, definitely worth checking.

Reply:
Ah! Good catch!

Reply:
About the other cases, they're correct already.

```php
		$site_url            = home_url( '', 'https' );
		$job_endpoint_url    = rest_url( '/wpjm-internal/v1/promoted-jobs/' . $post_id, 'https' );
		$job_endpoint_url    = substr( $job_endpoint_url, strlen( $site_url ) );
		$verify_endpoint_url = rest_url( '/wpjm-internal/v1/promoted-jobs/verify-token', 'https' );
		$verify_endpoint_url = substr( $verify_endpoint_url, strlen( $site_url ) );
```